lib/site_prism/exceptions.rb

```diff
@@ -11,7 +11,13 @@ def message
   end
 
   class NoSelectorForElement < StandardError; end
-  class TimeoutException < StandardError; end
+
+  class TimeoutException < StandardError
+    def message
+      "Timed out after #{super}s while waiting for block to evaluate as true."
```

Comment:
I don't think overriding `message` here is necessary or helpful, it would be better (IMO) to leave the message in the waiter class and just interpolate the time there. If I want to change the message in the future now I have to change two files instead of one

Reply:
For variable messages I'd tend to agree with you there. However this message (And one or two others), I've opted to hardcode and standardize - With a view to getting more of the suite under a generic style / feel.

This error message only triggers in one place (and is rescued in another), and this occurs only when there is a loop that times out. As such I wouldn't want this message to be altered, or mutable.